### PR TITLE
Remove models from base handler

### PR DESCRIPTION
**Pull request by seallard**
## Description
There is no reason to keep the models in the base handler as attributes any longer.

### Changed
- Removed all model attributes on the base handler class since they are unnecessary.

## Review
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by

### This [version](https://semver.org/) is a
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: tests/meta/upload/balsamic/test_balsamic.py [hunk 1]
```diff
@@ -24,9 +24,7 @@ def test_genotype_check_wgs_normal(balsamic_context: CGConfig):
     """Test a cancer case with WGS and normal sample that is Genotype compatible."""
     # GIVEN a balsamic case with WGS tag and a normal sample
     internal_id = "balsamic_case_wgs_paired_enough_reads"
-    case_obj: models.Family = balsamic_context.status_db.get_case_by_internal_id(
-        internal_id=internal_id
-    )
+    case_obj: Family = balsamic_context.status_db.get_case_by_internal_id(internal_id=internal_id)
```
Comment (henrikstranneheim) [inline tests/meta/upload/balsamic/test_balsamic.py:27]:
```suggestion
    case: Family = balsamic_context.status_db.get_case_by_internal_id(internal_id=internal_id)
```

File: tests/meta/upload/balsamic/test_balsamic.py [hunk 2]
```diff
@@ -39,9 +37,7 @@ def test_genotype_check_non_wgs_normal(balsamic_context: CGConfig):
     """Test a cancer case with no WGS sample that is not Genotype compatible."""
     # GIVEN a balsamic case with a normal sample, but no WGS tag
     internal_id = "balsamic_case_tgs_paired"
-    case_obj: models.Family = balsamic_context.status_db.get_case_by_internal_id(
-        internal_id=internal_id
-    )
+    case_obj: Family = balsamic_context.status_db.get_case_by_internal_id(internal_id=internal_id)
```
Comment (henrikstranneheim) [inline tests/meta/upload/balsamic/test_balsamic.py:40]:
```suggestion
    case: Family = balsamic_context.status_db.get_case_by_internal_id(internal_id=internal_id)
```

File: tests/meta/upload/balsamic/test_balsamic.py [hunk 3]
```diff
@@ -54,9 +50,7 @@ def test_genotype_check_only_tumour(balsamic_context: CGConfig):
     """Test a cancer case with only a tumour sample that is not Genotype compatible."""
     # GIVEN a balsamic case with only tumour sample
     internal_id = "balsamic_case_wgs_single"
-    case_obj: models.Family = balsamic_context.status_db.get_case_by_internal_id(
-        internal_id=internal_id
-    )
+    case_obj: Family = balsamic_context.status_db.get_case_by_internal_id(internal_id=internal_id)
```
Comment (henrikstranneheim) [inline tests/meta/upload/balsamic/test_balsamic.py:53]:
```suggestion
    case: Family = balsamic_context.status_db.get_case_by_internal_id(internal_id=internal_id)
```

File: tests/store/conftest.py [hunk 4]
```diff
@@ -149,9 +149,7 @@ def fixture_microbial_store(
         application_version = base_store.get_application_by_tag(
             sample_data["application"]
         ).versions[0]
-        organism = base_store.Organism(
-            internal_id=sample_data["organism"], name=sample_data["organism"]
-        )
+        organism = Organism(internal_id=sample_data["organism"], name=sample_data["organism"])
```
Comment (henrikstranneheim) [inline tests/store/conftest.py:152]: Type hint

File: tests/store_helpers.py [hunk 5]
```diff
@@ -456,7 +456,7 @@ def ensure_case_from_dict(
     ):
         """Load a case with samples and link relations from a dictionary."""
         customer_obj = StoreHelpers.ensure_customer(store)
-        case_obj = store.Family(
+        case_obj = Family(
```
Comment (henrikstranneheim) [inline tests/store_helpers.py:459]:
```suggestion
        case = Family(
```

**Review by henrikstranneheim — APPROVED** [review]
Great clean up 👍

File: tests/store/conftest.py (hunk 4 above)
Comment (seallard) [reply tests/store/conftest.py:152]: Added it, but I think it makes the code less readable in this instance

**Pull request merged**